.github/workflows/build-mudlet.yml

```diff
@@ -346,7 +346,7 @@ jobs:
       shell: bash
 
     - name: (macOS) Prep binary for running tests
-      if: matrix.run_tests == 'true' && runner.os == 'macOS'
+      if: matrix.run_tests == 'true' && runner.os == 'macOS' && env.UPLOAD_FILENAME
```

Comment:
Is that sufficient to determine that `UPLOAD_FILENAME` exists and is not an empty string in GH Actions' Yaml interpreter?

Reply:
It will not just exist or not exist for us, so it's OK for what it does.

Reply:
Okay then.